### PR TITLE
Add Uroria Network

### PR DESCRIPTION
**Pull request by julian-siebert**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by julian-siebert** [comment]
Our server already released its alpha last winter but we want to release our beta next month.
The ".de" and ".net" domain is currently not available but will get online before we release our beta.

File: minecraft_servers/uroria/manifest.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "uroria",
+  "nice_name": "Uroria Network",
+  "direct_ip": "play.uroria.com",
+  "server_wildcards": [
+    "%.uroria.com",
+    "%.uroria.de",
+    "%.uroria.net"
```
Comment (JakobBruening) [inline minecraft_servers/uroria/manifest.json:8]: Remove domains that are not available.

File: minecraft_servers/uroria/manifest.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  ],
+  "supported_languages": [
+    "de",
+    "lt",
+    "en",
+    "fr",
+    "la"
+  ],
+  "social": {
+    "web": "https://uroria.com",
```
Comment (JakobBruening) [inline minecraft_servers/uroria/manifest.json:18]: Fix or remove

Bildschirmfoto 2023-06-16 um 10 35 01
Bildschirmfoto 2023-06-16 um 10 34 57

File: minecraft_servers/uroria/manifest.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "instagram": "urorianetwork",
+    "discord": "https://discord.com/invite/RGX8rPshFG",
+    "twitter": "UroriaNetwork",
+    "youtube": "https://yt.uroria.com"
```
Comment (JakobBruening) [inline minecraft_servers/uroria/manifest.json:22]: Custom links are only supported for partner servers due to security reasons.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/uroria/manifest.json (hunk 3 above)
Comment (julian-siebert) [reply minecraft_servers/uroria/manifest.json:22]: Well I've removed them already

**Pull request closed without merging**